tests/node/test_native.py

```
@@ -23,25 +24,23 @@ def test_native_run():
+    with tmpdir_manager() as dir1, tmpdir_manager() as dir2:
+        tasks = [
+            RosettaCmdTask(cmd=["cmd1"], base_dir=dir1, task_label="label1"),
+            RosettaCmdTask(cmd=["cmd2"], base_dir=dir2, task_label="label2"),
+        ]
 
-    # Assert that the Parallel function was called with the correct arguments
-    # mock_parallel.assert_called_once_with(n_jobs=2, verbose=100)
+        # Mock the Parallel and delayed functions
+        with patch(
+            "joblib.Parallel", side_effect=lambda *args, **kwargs: [mock_execute(task, lambda x: x) for task in tasks]
+        ) as mock_parallel:
```

Ruff

Local variable `mock_parallel` is assigned to but never used

Remove assignment to unused variable `mock_parallel`

(F841)